public/docs/_examples/dependency-injection/dart/lib/heroes/mock_heroes.dart

```diff
@@ -0,0 +1,15 @@
+// #docregion
+import "hero.dart";
+
+List<Hero> HEROES = [
+  new Hero()..id = 11..isSecret = false..name = "Mr. Nice",
```

Comment:
Yes, please. In fact, be sure to always run dartfmt before uploading your changes.